python/src/uagents/agent.py

```diff
@@ -42,9 +42,16 @@
 from uagents.envelope import EnvelopeHistory, EnvelopeHistoryEntry
 from uagents.mailbox import MailboxClient
 from uagents.models import ErrorMessage, Model
-from uagents.network import InsufficientFundsError, get_almanac_contract, get_ledger
+from uagents.network import (
+    InsufficientFundsError,
+    get_almanac_contract,
+    get_ledger,
+)
 from uagents.protocol import Protocol
-from uagents.registration import AgentRegistrationPolicy, DefaultRegistrationPolicy
+from uagents.registration import (
```

Comment:
Are you using the most recent ruff version?
I think they changed import formatting at some point.

Reply:
Updated, thanks!